addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -794,6 +794,19 @@ def test_cash_rounding_amount_total_rounded(self):
             self.assertEqual(move.tax_totals['amount_total'], 544.7)
             self.assertEqual(move.tax_totals['amount_total_rounded'], 545)
 
+            # now confirm the invoice and also change the rounding method from HALF-UP to down
+            move.action_post()
+            move.invoice_cash_rounding_id.rounding_method = 'DOWN'
+            move._compute_tax_totals()
```

Comment:
I feel like it's weird to recompute the totals of an invoice after it has been validated. I'll double-check with tsb to be sure that's what we want.